xclim/indices/_conversion.py

```diff
@@ -695,3 +697,143 @@ def wind_chill_index(
 
     W.attrs["units"] = "degC"
     return W
+
+
+@declare_units(tasmin="[temperature]", tasmax="[temperature]")
+def potential_evapotranspiration(
+    tasmin: xr.DataArray, tasmax: xr.DataArray, tas: xr.DataArray, method: str = "BR65"
```

Comment:
Annotation and default values : Not all methods need all variables : variables not needed by some of the methods should default to None and have the `Optional[xr.DataArray` annotation. (See `relative_humidity` for example).

Same below: please mention in the parameters' doc which method needs which variable (and at which frequency).

Reply:
+1 for this comment.   Technically I think I should be able to calculate Thornwaite with only `tas`  with something like:
`atmos.potential_evapotranspiration(tas=tmC, method='TW48')`